corehq/apps/app_manager/util.py

```diff
@@ -409,7 +409,7 @@ def get_casedb_schema(form):
     This lists all case types and their properties for the given app.
     """
     app = form.get_app()
-    base_case_type = form.get_module().case_type
+    base_case_type = form.get_module().case_type if form.requires_case() else None
```

Comment:
backing up though, that is sort of the whole point of (or at least one of the main points of) the data dictionary, that it could then be used to add richness to vellum's knowledge of case properties. I think we're just not quite ready yet